mysql/004-failover.md

```diff
@@ -24,7 +24,7 @@ In order to keep this tutorial simple, we are going to delete the MySQL pod and
 
 Now, you can see the mount was removed but the location is still the same.
```

Comment:
The CLI doesn't show that the volume was detached so I'm not sure what you mean by this.

Reply:
This is the output I get: 
![image](https://user-images.githubusercontent.com/5175537/90128067-270c4000-dd5e-11ea-93cb-3a8bdaee4a61.png)

Is the pod fully deleted in your case? Maybe I should add that in the process.

Reply:
Ah right you are, I think I just clicked through too fast.